test/roo/test_excelx.rb

```diff
@@ -243,11 +243,11 @@ def test_bug_simple_spreadsheet_time_bug
 
   def test_simple2_excelx
     with_each_spreadsheet(name: "simple_spreadsheet", format: :excelx) do |workbook|
-      assert_equal [:numeric_or_formula, "yyyy\\-mm\\-dd"], workbook.excelx_type("A", 4)
-      assert_equal [:numeric_or_formula, "#,##0.00"], workbook.excelx_type("B", 4)
```

Comment:
> It looks like all of the code in this gem just discarded the array and took the last element.

It is not really the case and as you may observer in this test case it is an expected behavior.

https://github.com/roo-rb/roo/blob/8f65109c6728dd77f62b0a5c68dd42ab207dd914/lib/roo/excelx/cell/base.rb#L43

Excelx::Cell::Base does not use `Array#last` to discard `:numeric_or_formula`